cmd/kube-scheduler/app/options/options.go

```diff
@@ -43,7 +43,8 @@ import (
 	"k8s.io/component-base/logs"
 	"k8s.io/component-base/metrics"
 	"k8s.io/klog/v2"
-	kubeschedulerconfigv1beta1 "k8s.io/kube-scheduler/config/v1beta1"
+	kubeschedulerconfigv1beta2 "k8s.io/kube-scheduler/config/v1beta2"
```

Comment:
let's take the opportunity to change this alias to `configv1beta2`. We are in the kube-scheduler folder, so no need to say that again.

Reply:
can you still do this please?

Reply:
Do this please